systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -450,6 +504,19 @@ void testMirrorMaker2TlsAndScramSha512Auth() {
 
     @ParallelNamespaceTest
     @Tag(COMPONENT_SCALING)
+    @TestDoc(
+        description = @Desc("Verifies scaling MM2 up and down (including scale-to-zero)."),
+        steps = {
+            @Step(value = "Deploy source and target Kafka clusters.", expected = "Clusters are ready."),
+            @Step(value = "Deploy MM2 with initial replica count.", expected = "MM2 starts successfully."),
+            @Step(value = "Scale MM2 up and verify observedGeneration and pod names.", expected = "Pods increase and new pods are named correctly."),
+            @Step(value = "Scale MM2 down to zero and verify pod removal.", expected = "All MM2 pods are removed, observedGeneration increases, status is Ready."),
+            @Step(value = "Wait until MM2 status URL is null.", expected = "Status reflects shutdown.")
```

Comment:
Maybe it would be better to mention actual values for replicas to better describe the actual scenario?